pkgs/tools/text/sift/deps.nix

```diff
@@ -0,0 +1,35 @@
+{ fetchgit }:
+[
+  {
+    goPackagePath = "golang.org/x/crypto";
+    src = fetchgit {
+      url = "https://go.googlesource.com/crypto";
+      rev = "575fdbe86e5dd89229707ebec0575ce7d088a4a6";
+      sha256 = "1kgv1mkw9y404pk3lcwbs0vgl133mwyp294i18jg9hp10s5d56xa";
+    };
+  }
+  {
+    goPackagePath = "golang.org/x/crypto";
```

Comment:
x/crypto is duplicated in this deps file -- might want to check elsewhere for similar issues